### ISSUE TITLE
[TRACKING] WG-Green Review Logo 

### ISSUE THREAD

**Issue by AntonioDiTuri** [cncf-tags/green-reviews-tooling#71]
While writing the [article-draft](https://docs.google.com/document/d/1_MnYd1hHhRysHsWCfaSuNpAEc09GuBF61Qt0rCqJDrk/edit) about the project update of the green review the need for a WG-logo came up. 

@leonardpahlke suggested it might be something to request to CNCF after KubeCon.

Any contribution or idea how this logo should look like is welcome from the community :)

**Comment by nikimanoledaki** [comment]
@leonardpahlke @guidemetothemoon is it possible to make a request to the CNCF through the service desk to get help with creating a logo and other "branded" graphics?

**Comment by leonardpahlke** [comment]
Yes 💯 - @nikimanoledaki @AntonioDiTuri could you fill out this document with information about the logo. https://docs.google.com/document/d/1t0-EzbwJBVUdIKrgR_qv-IZo-_O_NibjaL2BouGVxAY/edit?usp=sharing

**Comment by leonardpahlke** [comment]
Let me know as soon as it is filled out, so i can review and submit it 👍

**Comment by leonardpahlke** [comment]
👍 i will open a ticket
